libs/networking/dune

```diff
@@ -0,0 +1,22 @@
+(library
+ (public_name http)
```

Comment:
better networking for the name, to match the directory
(and so that later we can add helpers for other networking stuff, not just http)